### PR TITLE
Show count of bids and link to edge browser in reviewer summary cell

### PR DESCRIPTION
**Pull request by zbialecki**
Fixes https://github.com/openreview/openreview/issues/1916

### CODE REVIEW

**Comment by melisabok** [comment]
Running test_eccv_conference.py the reviewers/ACs are not being shown because there are no assignment groups, we should list them anyway, it is very important to see the status of the bids before the assignment.

Screen Shot 2020-03-03 at 1 28 13 PM

**Comment by melisabok** [comment]
This line is iterating through all the area chair group ids, we can start the for using the area chair members and then search the ids:

https://github.com/openreview/openreview-py/blob/master/openreview/conference/templates/programchairWebfield.js#L1046

This will fix another issue that is when an AC/Reviewer doesn't have assigned papers is not being shown.

**Comment by zbialecki** [comment]
OK, I'll make that change.

**Comment by zbialecki** [comment]
Now the reviewer tab and the AC tab will be populated regardless if they have been assigned papers. In addition the AC tab will also now show counts of completed recommendations similar to the counted bids in the reviewers tab.

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -1309,6 +1336,21 @@ var updateReviewerContainer = function(paperNumber) {
...
+  var invitationId = invGroup + '/-/' + invName;
+  var referrerUrl = '/group' + location.search + location.hash;
+
+  // Right now this is only showing bids, elmo scores, and conflicts as the
+  // other scores invitations + labels are not available in the PC console
+  return '/edge/browse' +
+    '?start=' + invitationId + ',tail:' + profileId +
+    '&traverse=' + invitationId +
+    '&browse=' + invitationId +
+    ';' + invGroup + '/-/ELMo_Score' +
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js]:
```suggestion
    ';' + invGroup + '/-/Affinity_Score' +
```

Comment (melisabok) [reply openreview/conference/templates/programchairWebfield.js]: We should parametrize this, ECCV will use Affinity_Score and TPMS_Score.

**Comment by melisabok** [comment]
Running test_eccv_conference.py I can see the reviewers but not the area chairs.

**Comment by melisabok** [comment]
When I try to see the bids for for Reviewer ECCV_Three, it has 3 bids, I only see two papers in the edge browser:

Screen Shot 2020-03-03 at 5 51 14 PM


Now I understand your change in the openreview repo to show the Bid in the first column.

**Comment by zbialecki** [comment]
> When I try to see the bids for for Reviewer ECCV_Three, it has 3 bids, I only see two papers in the edge browser:

That means that the paper did not match the invitation query specified by the edge invitation (usually because it was withdrawn).

**Comment by zbialecki** [comment]
The ACs tab should be working correctly now. It also seems like the Bid invitation was set up with the wrong `reply.content.tail.type` so I fixed that too.

**Comment by zbialecki** [comment]
Note: to see the AC recommendations count and edge browser links you have to add values to 2 of the  constants at the top of the file:
```
var RECOMMENDATION_NAME = 'Recommendation';
var SCORES_NAME = 'Affinity_Score';
```

File: openreview/conference/templates/programchairWebfield.js [hunk 2]
```diff
@@ -266,6 +273,49 @@ var postReviewerEmails = function(postData) {
...
+};
+
+var getAreaChairRecommendationCounts = function() {
+  if (!RECOMMENDATION_NAME) {
+    return $.Deferred().resolve({});
+  }
+
+  // Can't perform a groupBy query on signatures, so we have to get the full list
+  return Webfield.getAll('/edges', {
+    invitation: REVIEWERS_ID + '/-/' + RECOMMENDATION_NAME,
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js]:
```suggestion
    invitation: AREA_CHAIRS_ID + '/-/' + RECOMMENDATION_NAME,
```

File: openreview/conference/templates/programchairWebfield.js [hunk 3]
```diff
@@ -266,6 +273,49 @@ var postReviewerEmails = function(postData) {
   });
 };
 
+var getReviewerBidCounts = function() {
+  if (!BID_NAME) {
+    return $.Deferred().resolve({});
+  }
+
+  return Webfield.getAll('/edges', {
+    invitation: REVIEWERS_ID + '/-/' + BID_NAME,
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js]: area chairs have a different invitation to get the edges, it starts with the AREA_CHAIRS_ID

Comment (zbialecki) [reply openreview/conference/templates/programchairWebfield.js]: Ah ok, right now I am not showing the area chair bids, but I could get those here as well

**Pull request merged**